### ISSUE TITLE
french stop list seems not to work

### ISSUE THREAD

**Issue by colinallen** [inpho/topic-explorer#159]
Common words such as les, ils autre, showing up after application of stoplist

**Comment by JaimieMurdock** [comment]
We tracked this down to the french stoplist being stemmed, rather than unstemmed.

Moving forward, we may want to start building our own stoplist collections, which may enable us to cut our `nltk` dependency. Having a workshop on stoplists at an ACL meeting could be a good idea for eventual community building.

**Comment by JaimieMurdock** [comment]
The French stoplist is stemmed. Right now we are using the nltk stoplists. Mallet seems to have some better ones, and we may be able to find others to include with the explorer.

Do we want to include stoplist curation as something we do or should we continue to delegate that responsibility to other projects? If we continue to delegate, should we select a different authority than nltk?

**Comment by colinallen** [comment]
I'm not sure we want to get into the business of curating stoplists, although maybe we can accept submissions at Github and explain to users how to work with them? 

There's pretty good one for French at http://snowball.tartarus.org/algorithms/french/stop.txt but it would require reformatting for our purposes since it has some comment fields.   Maybe we should make the stoplist importer a bit more sophisticated by having it ignore  everything past any non-word character that appears on a line.  That would handle standard # comments as well as unique formats such the one above.

Could we also provide an http stoplist option, so that we would not need to copy this, but simply call pycurl to download and clean a local copy for the user the first time they specify it?

**Comment by JaimieMurdock** [comment]
I am going to remove the `--lang` flag from the printed help while leaving the feature in as undocumented. 

I think that we can expand the documentation for `--stopword-file` to detail how to use that in conjunction with another stoplist. I don't think we should get into the business of curation either, and I think we should also not add more support for comments, HTTP or S3 stoplists right now.

Basically, abandon the feature so we have something easier to maintain. Recommend use of `--stopword-file` to help.

This is done in the documentation branch, will be merged on the next beta release.
